### ISSUE TITLE
Creating wiki page triggers 500 error

### ISSUE THREAD

**Issue by klizas** [gitlabhq/gitlabhq#10320]
I'm trying to create wiki page for repository, and I'm getting 500 error. Here is entry from production.log:

 Parameters: {"utf8"=>"✓", "authenticity_token"=>"[FILTERED]", "wiki"=>{"title"=>"home", "format"=>"markdown", "content"=>"sdasd", "message"=>""}, "namespace_id"=>"kemdu", "project_id"=>"fotomuziejus"}
Completed 500 Internal Server Error in 102ms (ActiveRecord: 10.6ms)

NoMethodError (undefined method `username' for #<Group:0x00000007a3a898>):
  app/models/project_wiki.rb:158:in `default_message'
  app/models/project_wiki.rb:152:in `commit_details'
  app/models/project_wiki.rb:91:in `create_page'
  app/models/wiki_page.rb:183:in `save'
  app/models/wiki_page.rb:143:in `create'
  app/services/wiki_pages/create_service.rb:6:in `execute'
  app/controllers/projects/wikis_controller.rb:58:in `create'
  lib/gitlab/middleware/go.rb:16:in `call'

```
System information
System:		Debian 7.10
Current User:	git
Using RVM:	no
Ruby Version:	2.1.8p440
Gem Version:	2.5.1
Bundler Version:1.10.6
Rake Version:	10.5.0
Sidekiq Version:4.0.1

GitLab information
Version:	8.7.1
Revision:	93187cb
Directory:	/opt/gitlab/embedded/service/gitlab-rails
DB Adapter:	postgresql
URL:		https://gitlab.kemdu.lt
HTTP Clone URL:	https://gitlab.kemdu.lt/some-group/some-project.git
SSH Clone URL:	git@gitlab.kemdu.lt:some-group/some-project.git
Using LDAP:	no
Using Omniauth:	no

GitLab Shell
Version:	2.7.2
Repositories:	/var/opt/gitlab/git-data/repositories
Hooks:		/opt/gitlab/embedded/service/gitlab-shell/hooks/
Git:		/opt/gitlab/embedded/bin/git
```

**Comment by vicvega** [comment]
It's a know bug of `8.7`.
It's been fixed [here](https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/3924)

**Comment by connorshea** [comment]
This was fixed in 8.7.2 :)

https://about.gitlab.com/2016/05/03/gitlab-8-dot-7-dot-2-released/

**Comment by connorshea** [comment]
Hi! We’re closing our issue tracker on GitHub so we can focus on the GitLab.com issue tracker and respond to issues faster.

If you think this is still an issue I encourage you to open it [in our issue tracker on GitLab.com](https://gitlab.com/gitlab-org/gitlab-ce/issues). You can log in on GitLab.com using your GitHub account if you'd like to contribute an issue.
